.github/workflows/warn-if-behind-dev.yml

```
@@ -0,0 +1,45 @@
+      - name: Fetch dev branch
+        run: git fetch origin dev
+
+      - name: Count commits PR branch is behind dev
+        id: behind
+        run: |
+          BEHIND_COUNT=$(git rev-list --count HEAD..origin/dev)
+          echo "behind_count=$BEHIND_COUNT" >> $GITHUB_OUTPUT
+
+      - name: Warn author if behind dev branch
+        if: ${{ steps.behind.outputs.behind_count > 0 }}
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)